src/liberty/libertyProject.ts

```diff
@@ -208,6 +208,7 @@ export class ProjectProvider implements vscode.TreeDataProvider<LibertyProject>
 	}
 
 	public getTreeItem(element: LibertyProject): vscode.TreeItem {
+		element.tooltip = element.path;
```

Comment:
@TrevCraw Addressed the review comment

Reply:
Please remove the original line, as the intent of adding to the constructor is so that the `tooltip` field is set once and does not get reassigned a new value when `getTreeItem()` is called.